src/Microsoft.DotNet.Cli.Sln.Internal/LocalizableStrings.cs

```diff
@@ -2,6 +2,14 @@
 {
     internal class LocalizableStrings
     {
+        // {0} is the line number
+        // {1} is the error message details
+        public const string ErrorMessageFormatString = "Invalid format in line {0}: {1}";
```

Comment:
If you do this now, we will need to push this through Loc again.

@srivatsn Does this meet the bar for loc?